### ISSUE TITLE
client-certificate and client-key have identical definitions.

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#13603]
The client-certificate and client key command line options in kube-api-server/app/server.go seem to have identical definitions.

```
236     fs.StringVar(&s.KubeletConfig.CertFile, "kubelet-client-certificate", s.KubeletConfig.CertFile, "Path to a client key file for TLS.")
237     fs.StringVar(&s.KubeletConfig.KeyFile, "kubelet-client-key", s.KubeletConfig.KeyFile, "Path to a client key file for TLS.")

 ```

**Comment by roberthbailey** [comment]
This looks like a typo. They certainly have different meanings. Can you send a PR to fix the description for the client certificate flag?

**Comment by jayunit100** [comment]
hi -- ok, #13672 fixes this.

**Comment by gmarek** [comment]
Fixed by  #13672 according ot @jayunit100
